components/VideoPlayer.vue

```diff
@@ -90,6 +91,19 @@ export default Vue.extend({
         );
         // Register player events
         this.player.addEventListener('error', this.onPlayerError);
+        // Subscribe to Vuex actions
+        this.$store.subscribe((mutation, _state: AppState) => {
+          switch (mutation.type) {
+            case 'playbackManager/PAUSE_PLAYBACK':
+              (this.$refs.videoPlayer as HTMLVideoElement).pause();
+              break;
+            case 'playbackManager/UNPAUSE_PLAYBACK':
+              (this.$refs.videoPlayer as HTMLVideoElement).play();
+              break;
+            case 'playbackManager/RESET_CURRENT_TIME':
+              (this.$refs.videoPlayer as HTMLVideoElement).currentTime = 0;
+          }
+        });
```

Comment:
when any of the mutations are called, it is erroring out:
`Cannot read property 'pause' of undefined`

Reply:
I can't seem to reproduce it on either Chromium or Firefox. I added null checks, so we don't trigger it accidentally.

Pause works here, but the close/full button had some issues due to a div being in front of them. So I added some pointer-event stuff in CSS to fix it. That fixes it for me.

Reply:
Strange, I've retested and it all seems to be working now.